client/my-sites/theme/main.jsx

```diff
@@ -187,9 +178,17 @@ const ThemeSheet = React.createClass( {
 		return null;
 	},
 
+	getPreviewAction() {
+		const { preview } = this.props.options;
+		return () => {
+			this.props.setPreviewOptions( this.props.defaultOption, this.props.secondaryOption );
+			return preview.action( this.props.theme );
+		};
+	},
```

Comment:
Isn't it possible to simplify this method by dropping the wrapping closure and putting in everything flat,

```js
getPreviewAction() {
	const { preview } = this.props.options;
	this.props.setPreviewOptions( this.props.defaultOption, this.props.secondaryOption );
	preview.action( this.props.theme );
}
```

and dropping the parentheses after `this.getPreviewAction` in the `onClick` handler below?